.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: go
+go:
+  - "1.12.x"
+script:
+  - make lint
```

Comment:
I was about to write 'how about test?' but this repo doesn't have tests 🐱

Reply:
i think so too.